sdk/python/test-requirements.txt

```diff
@@ -5,3 +5,5 @@ tox >= 3.9.0
 flake8 >= 4.0.0
 types-python-dateutil >= 2.8.19.14
 mypy >= 1.5
+urllib3>=2.2.2 # not directly required, pinned by Snyk to avoid a vulnerability
```

Comment:
Instead of modifying a upstream file, I think it is better to exclude these directories in .snyk file as in https://github.com/openshift/kubernetes-sigs-lws/blob/main/.snyk

Reply:
Yea, sorry this was automated by sync. I did not actually open this up.

I tried to push this upstream but I wasn't able to figure out quickly how to persist these updates.